pages/docs/components/layout/container.mdx

```diff
@@ -57,13 +57,16 @@ To set the size of a Container, use the `maxW` attribute.
 ## Centering the children
 
 In some cases, the width of the content can be smaller than the container's
-width, you can use the `centerContent` prop to center the content.
+width. You can use the `centerContent` prop to center the content. It renders
+flexbox with vertical direction and `align-items` set to `center`.
```

Comment:
It renders *a* flexbox with `direction` set to `vertical` and `alignItems` set to `center`.